_data/sites.json

```diff
@@ -1969,6 +1969,20 @@
         ]
     },
 
+    {
+        "name": "Discourse",
+        "url": "https://www.discourse.org/",
+        "difficulty": "impossible",
+        "notes": "There seems to be no standard way of handling account deletions on discourse-hosted forums.",
+        "domains": [
+            "community.netlify.com",
+            "forum.level1techs.com",
+            "forum.rclone.org",
+            "forum.restic.net",
+            "forum.syncthing.net"
```

Comment:
Here's mine:
```suggestion
            "forum.syncthing.net",
            "https://community.containo.us",
            "https://forum.cloudron.io",
            "https://forum.duplicati.com",
            "https://discourse.gnome.org",
            "https://discourse.joplinapp.org"
```

Reply:
Can you please suggest them without the leading `https://` and preferably sorted?